specification/compute/Compute.Management/client.tsp

```diff
@@ -0,0 +1,40 @@
+import "./main.tsp";
+import "@azure-tools/typespec-client-generator-core";
+
+using Azure.ClientGenerator.Core;
+using Microsoft.Compute;
+
+@@clientName(NetworkApiVersion.`2020-11-01`, "TwoThousandTwenty1101", "go");
+@@clientName(NetworkApiVersion.`2022-11-01`, "TwoThousandTwentyTwo1101", "go");
```

Comment:
I'm not sure what are these?

Reply:
Just a renaming. It is used in one of your API.